guide/voice/the-basics.md

```diff
@@ -0,0 +1,123 @@
+# The Basics
+
+Now that you have your dependencies installed, you can start using voice!
+
+## Joining voice channels
+
+Joining a voice channel is easy, here's a common example:
+
+```js
+client.on('message', async message => {
+	// Join the same voice channel of the author of the message
+	if (message.member.voice.channel) {
+		const connection = await message.member.voice.channel.join();
+	}
+});
+```
+
+If you try to join a channel the bot is already in, nothing will happen so we don't need to worry about running this method repeatedly.
+
+::: tip
+In the above example we accessed a `VoiceState` through `member.voice`. A voice state exists for members connected to a voice channel, and it tells you which channel they're connected to, if the member is speaking, and whether they're deafened or muted. Consult the documentation for a full description of what voice states can do!
+:::
+
+## Playing audio
+
+Playing audio is also simple. A `StreamDispatcher` is created for you to control and monitor the playback of your audio over the voice channel.
+
+A VoiceConnection can only ever have at most one dispatcher. If you try to play another stream while audio is already playing, the existing stream will be destroyed and the new audio will begin to play shortly after.
+
+```js
+// Create a dispatcher
+const dispatcher = connection.play('audio.mp3');
+
+dispatcher.on('start', () => {
+	console.log('audio.mp3 is now playing!');
+});
+
+dispatcher.on('finish', () => {
+	console.log('audio.mp3 is finished playing!');
```

Comment:
"has finished" instead of "is finished" sounds better to me here, but again personal preference.